CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingImageView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  OnboardingImageView.swift
+//  GiniCapture
+//  Created by Nadya Karaban on 08.06.22.
+//
+
+import UIKit
+
+class OnboardingImageView: UIView, OnboardingIllustrationAdapter {
```

Comment:
Just an observation: if the adapter itself is a view, then it has to be initialised when configuring the sdk and will live until the sdk is stopped even if it is never used.

To prevent that the adapter itself should be a simple object that creates (in `injectedView()`) and destroys (in `onDeinit()`) the injected view. The adapter then has a minimal memory footprint and ensures the view exists only when required.

Reply:
Thank you @a-szotyori for your comment I will keep in mind and update it